webpack-plugin/src/dynamic-resolve.js

```diff
@@ -2,6 +2,10 @@
 import loaderUtils from 'loader-utils';
 import path from 'path';
 
+// backslashes need to be duplicated, because we loose them all otherwise
+// when turned into a string in the return statement
+const adjustBackSlashes = (p) => p.replace(/\\/g, '\\\\');
```

Comment:
Can we put this in `src/utils/` and add unit test? I'm not 100% sure what the expected behaviour is to be honest.